README.md

```diff
@@ -49,20 +33,23 @@ $ python setup.py install
 
 ## Documentation
 
-The python documentation is built using [sphinx](http://sphinx-doc.org/) and
-[numpydoc](https://pypi.python.org/pypi/numpydoc).
-
 ```
-$ make doc
+from bdbcontrib import Population
+help(Population)
+foo = Population(...)
+foo.help("plot")
 ```
 
-## Use
+## Contributing
 
-If you want the Python API, `import bdbcontrib.plot_utils`, &c.
+This repository is currently using "Light Review" from
+http://tinyurl.com/probcomp-review-standards
 
-If you are using the bayeslite shell, load the bayeslite shell
-commands with:
-
-```
-.hook /path/to/bdbcontrib/hooks/hook_plotting.py
-```
+Our compatibility aim is to work on probcomp machines and members'
+laptops, and to provide scripts and instructions that make it not too
+hard to re-create our environments elsewhere. Polished packaging,
+broad installability without much work, etc. are anti-goals, because
```

Comment:
I find the terms "anti-goal" (and at one point "anti-priority") unnecessary. We should just say "is not a goal" or "not a priority". Moreover, it is not true that proper packaging and instability are incompatible with our core mission of probabilistic computing as the statement implies -- they are just not the highest priority right now.